package.json

```diff
@@ -358,7 +360,7 @@
     "webpack-dev-server": "4.7.4",
     "webpack-hot-middleware": "2.25.1",
     "webpack-merge": "5.7.3",
-    "workspace-tools": "0.27.0",
+    "workspace-tools": "0.29.1",
```

Comment:
monosize needs this package or why the bump ? I'm asking because in future lage and tools related to it will no longer be part of this repo

Reply:
yup, `monosize` relies on a newer version. There is one more usage of `workspace-tools` in `scripts/generators/rename-package.ts`. Once it and current bundle size tool will be removed from the repo, we can drop this dependency.